nucliadb_node/Cargo.toml

```diff
@@ -96,6 +96,7 @@ hostname = "0.3"
 username = "0.2"
 md5 = "0.7"
 lazy_static = "1.4.0"
+lru = "0.12.1"
```

Comment:
This was already in our dependency tree (via tantivy) so this should be pretty lightweigh.